docs/repairs-api/intro.md

```diff
@@ -220,6 +222,24 @@ The following is a JSON schema for the allowed appointment slots data structure:
 }
 ```
 
+### Repair days Mapping
+This configuration is for the mapping between a repair's priority and the number of days in which it is expected to be completed. This mapping is used when reporting back to the user how long a repair is expected to take.
```

Comment:
This change includes
> expected to be completed

The phrasing in the prototype is:
> We will assess your request and a repair will be scheduled within the next 30 days.

I suspect the latter is correct, please confirm with Product Owner.
Documentation should be updated to reflect appropriately

Reply:
Updated text to 'This configuration is for the mapping between a repair's priority and the number of days in which it is expected to be scheduled and completed. ' after conversation with Sonia.